.buildkite/pipeline.yml

```diff
@@ -585,6 +585,7 @@ steps:
       RUBY_TEST_VERSION: '2.0'
     concurrency: 4
     concurrency_group: 'ruby/integrations-maze-runner-tests'
+    if: "false"
```

Comment:
It's the same effect and I'm not overly bothered, but Buildkite has a `skip` that may be better - for example if we wanted to audit skipped tests across the pipelines it would be easier to find.  This is one from Android:
```    
skip: Temporarily disabled due to flakiness with BrowserStack tunneling
```

You may also want to consider a "soft fail", where the test is still run but doesn't cause the pipeline to have an overall fail.  It's wasted energy, perhaps, but the result is more in your face.  I guess it depends how much you trust yourself to work fix the failures in time.
```
    soft_fail:
      - exit_status: "*"
```

Reply:
Ah good to know; I didn't see `skip` in the docs. Soft fail sounds good, I'll do that instead